### PR TITLE
fix(learn): update helmet and bcrypt instructions so we can remove them from boilerplates

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/39720

### CODE REVIEW

**Comment by Sky020** [comment]
Tom, is this a work-in-progress?

I would have expected all of the information that was in the boilerplate to be ported over to the relevant lessons (eg):
```
/** 5) Avoid inferring the response MIME type - `helmet.noSniff()` */

// Browsers can use content or MIME sniffing to override response `Content-Type`
// headers to guess and process the data using an implicit content type.
// While this can be convenient in some scenarios, it can also lead to
// some dangerous attacks.
// This middleware sets the X-Content-Type-Options header to `nosniff`,
// instructing the browser to not bypass the provided `Content-Type`.
```

**Comment by moT01** [comment]
Nice catch - I think that sounds a little better, I'll update the PR to the main repo.

File: curriculum/challenges/english/09-information-security/information-security-with-helmetjs/install-and-require-helmet.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ Helmet helps you secure your Express apps by setting various HTTP headers.
 ## Instructions
 <section id='instructions'>
 
+All your code for these lessons goes in the `myapp.js` file between the lines of code we have started you off with. Do not change or delete the code we have added for you.
```
Comment (Sky020) [inline curriculum/challenges/english/09-information-security/information-security-with-helmetjs/install-and-require-helmet.md:17]:
```suggestion
All your code for these lessons goes in the `myApp.js` file between the lines of code we have started you off with. Do not change or delete the code we have added for you.
```

File: curriculum/challenges/english/09-information-security/information-security-with-helmetjs/understand-bcrypt-hashes.md [hunk 2]
```diff
@@ -15,6 +15,9 @@ BCrypt hashes will always looks like <code>$2a$13$ZyprE5MRw2Q3WpNOGZWGbeG7ADUre1
 ## Instructions
 <section id='instructions'>
 To begin using BCrypt, add it as a dependency in your project and require it as 'bcrypt' in your server.
+
+Remember that this is using a new boilerplate. Add all your code for these lessons in the `server.js` file between the code we have started you off with. Do not change or delete the code we have added for you.
```
Comment (Sky020) [inline curriculum/challenges/english/09-information-security/information-security-with-helmetjs/understand-bcrypt-hashes.md:19]:
```suggestion
Add all your code for these lessons in the `server.js` file between the code we have started you off with. Do not change or delete the code we have added for you.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Minor filename mention change.

Second comment I leave up to you, but the description does already mention the new project starter code. If you think this is a useful statement to ensure campers do not get confused, then dismiss this suggestion.

After this, this PR looks good.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**